pkg/activator/dedupe.go

```diff
@@ -86,7 +98,17 @@ func (a *dedupingActivator) dedupe(id revisionID, ch chan activationResult) {
 }
 
 func (a *dedupingActivator) activate(id revisionID) {
-	endpoint, status, err := a.activator.ActiveEndpoint(id.namespace, id.name)
+	logger := loggerWithRevisionInfo(a.logger, id.namespace, id.name)
```

Comment:
Seems like an overkill to just report the serving state. Is this an important thing to report?

Reply:
I simplified the code to report the serving state.